.gitignore

```diff
@@ -21,3 +21,4 @@ with_time_logs/
 *.json
 .hypothesis
 .array_api_tests_k_flag*
+internal_automation_tools/
```

Comment:
I have no idea, will remove.

Reply:
This is for the pr dashboard, horian mentioned that this folder should be in the .gitignore so I had added so that I dont accidently commit it to master. It was mentioned in a github discussion.

Reply:
Oh so are we leaving in or removing?